rollup.config.js

```diff
@@ -0,0 +1,50 @@
+const terser = require('rollup-plugin-terser').terser;
+const pkg = require('./package.json');
+
+const banner = `/*!
+ * ${pkg.name} v${pkg.version}
+ * ${pkg.homepage}
+ * (c) ${new Date().getFullYear()} Chart.js Contributors
+ * Released under the ${pkg.license} license
+ */`;
+
+module.exports = [
+	{
+		input: 'src/index.js',
+		output: {
+			name: 'ChartTypeSmith',
```

Comment:
You can (should?) remove `name` because this project doesn't export anything as a plugin would do.

Reply:
Done